public/pics/logo/logo.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" standalone="no"?>

```

Comment:
Maybe we should change the folder "pics" to "img" it is more common

Reply:
changed